python/metatensor-torch/metatensor/torch/atomistic/openmm_interface.py

```diff
@@ -0,0 +1,235 @@
+from typing import Iterable, List, Optional
+
+import torch
+
+from metatensor.torch import Labels, TensorBlock
+from metatensor.torch.atomistic import (
+    ModelEvaluationOptions,
+    ModelOutput,
+    System,
+    load_atomistic_model,
+)
+
+
+try:
+    import openmm
+    import openmmtorch
+    from NNPOps.neighbors import getNeighborPairs
+
+    HAS_OPENMM = True
+except ImportError:
+
+    class MLPotential:
+        pass
+
+    class MLPotentialImpl:
+        pass
+
+    class MLPotentialImplFactory:
+        pass
+
+    HAS_OPENMM = False
+
+
+def get_metatensor_force(
+    system: openmm.System,
+    topology: openmm.app.Topology,
+    path: str,
+    extensions_directory: Optional[str] = None,
+    forceGroup: int = 0,
+    atoms: Optional[Iterable[int]] = None,
+    check_consistency: bool = False,
+) -> openmm.System:
+
+    if not HAS_OPENMM:
+        raise ImportError(
+            "Could not import openmm and/or nnpops. If you want to use metatensor with "
+            "openmm, please install openmm-torch and nnpops with conda."
+        )
+
+    model = load_atomistic_model(path, extensions_directory=extensions_directory)
+
+    # Get the atomic numbers of the ML region.
+    all_atoms = list(topology.atoms())
+    atomic_numbers = [atom.element.atomic_number for atom in all_atoms]
+
+    if atoms is None:
+        selected_atoms = None
+    else:
+        selected_atoms = Labels(
+            names=["system", "atom"],
+            values=torch.tensor(
+                [[0, selected_atom] for selected_atom in atoms],
+                dtype=torch.int32,
+            ),
+        )
+
+    class MetatensorForce(torch.nn.Module):
+
+        def __init__(
+            self,
+            model: torch.jit._script.RecursiveScriptModule,
```

Comment:
```suggestion
            model,
```

We don't really need the types in `__init__`, and this one is private